src/open_samus_returns_rando/files/custom_scenario.lua

```diff
@@ -81,4 +81,12 @@ function Scenario.ShowText()
     Init.tBoxesSeen = textboxes_seen + 1
     GUI.LaunchMessage("#RANDO_STARTING_TEXT_" .. Init.tBoxesSeen, "Scenario.ShowText", "")
   end
+end
+
+function Scenario.LoadNewScenario(target_scenario, target_spawnpoint)
+  bWaitingForScenarioChange = true
+  Game.FadeOut(0.0)
+  Game.FadeOutStream(0.0 * 0.8)
+  Game.AddPSF(0.1, "Game.LoadScenario", "ssssi", "c10_samus", target_scenario, target_spawnpoint, "", 1)
+  Game.GetPlayer().vPos = V3D(-22800, 4450, 0)
```

Comment:
This probably shouldn't be included in the function if it's also being used for warp to start

Reply:
Oops. Doesn't change anyting logically but bad to have it there.
Removed.